specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databases.json

```diff
@@ -200,29 +200,38 @@
         },
         "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "Creates a database with minimum number of parameters.": {
-            "$ref": "./examples/CreateDatabaseMin.json"
+          "Creates a VCore database by specifying service objective name.": {
+            "$ref": "./examples/CreateVCoreDatabaseByServiceObjective.json"
           },
-          "Creates a VCore database with minimum number of parameters.": {
-            "$ref": "./examples/CreateVCoreDatabase.json"
```

Comment:
This example isn't referenced anymore. Should it be deleted?

Reply:
Nicely spotted. It was renamed, so the old file name can go away.

Reply:
I was just poking at the Avocado results 😝 The current error message isn't the most obvious though

Reply:
Thanks for that @nschonni